contacts-in-mautic.php

```diff
@@ -599,4 +599,4 @@ function bsfm_connect_mautic_username_password( $data, $anonymous ) {
 	$mautic_response['body'] = $body;
 
 	return $mautic_response;
-}
\ No newline at end of file
+}
```

Comment:
@nileshc-bsf Discard this change as this will prevent from releasing readme update.

Reply:
Okay, but here pipeline will fail, due to PHPCS.

Reply:
@nileshc-bsf  Do this change in the next bugfix / minor update. Let the PHPCS pipeline fail for this PR.

Reply:
@patilvikasj - Removed the changes.